pages/components/list/list.css

```diff
@@ -0,0 +1,82 @@
+.list-style-none {
+  list-style: none;
+  padding-inline-start: 0;
+}
+.inline-list {
+  display: flex;
+}
+.list-inline-item:not(:last-of-type) {
+  margin-right: 1.5rem;
+}
```

Comment:
I think these two are connected so maybe you could have used similar naming like for items you would have used inline-list-item or changed list name as list-inline. correct me if I am wrong.

Reply:
The first one is for the container, while the second class is for a list item. Hence, made different classes.